aten/src/ATen/native/native_functions.yaml

```diff
@@ -1935,6 +1935,36 @@
 - func: isclose(Tensor self, Tensor other, float rtol=1e-05, float atol=1e-08, bool equal_nan=False) -> Tensor
   variants: function, method
 
+- func: isin.Tensor_Tensor(Tensor elements, Tensor test_elements, bool assume_unique=False, bool invert=False) -> Tensor
+  variants: function
+  dispatch:
+    CPU, CUDA: isin
```

Comment:
This and the other functional variants are implemented via `at::` redispatch which suggests to me that, while correct, the dispatch entries here are too conservative.

You could bypass all of this faffing about by rewriting these as structured kernels; if you're not in a hurry I suggest trying it out https://github.com/pytorch/rfcs/blob/rfc-0005/RFC-0005-structured-kernel-definitions.md

Reply:
Cool, I'll try it out! This PR isn't really too high of a priority